frontend/src/components/aside/index.js

```diff
@@ -1,15 +1,15 @@
-
-import React from "react";
-import Courier from "../forms/Courier";
-import Stocks from "../forms/Stocks";
-import User from "../forms/User";
-import Customer from "../forms/Customer";
-import Product from "../forms/Product";
-import Order from "../forms/Order";
-import OrderDetails from "../forms/OrderDetails";
-import CourierInfo from "../forms/CourierInfo";
+import React from "react"
+import Courier from "../forms/Courier"
+import CourierInfo from "../forms/CourierInfo"
+import Customer from "../forms/Customer"
+import Login from "../forms/login"
+import Order from "../forms/Order"
+import OrderDetails from "../forms/OrderDetails"
+import Product from "../forms/Product"
+import Stocks from "../forms/Stocks"
+import User from "../forms/User"
```

Comment:
По рукам бы тебе надовать...
```suggestion
import React from "react";
import Courier from "../forms/Courier";
import CourierInfo from "../forms/CourierInfo";
import Customer from "../forms/Customer";
import Login from "../forms/login";
import Order from "../forms/Order";
import OrderDetails from "../forms/OrderDetails";
import Product from "../forms/Product";
import Stocks from "../forms/Stocks";
import User from "../forms/User";
```